nexus3/pkg/client/client.go

```diff
@@ -32,6 +34,25 @@ func NewClient(config Config) *Client {
 		config.Timeout = &defaultTimeout
 	}
 
+	var caCertPool *x509.CertPool
+	if *config.RootCAPath != "" {
```

Comment:
pointer for RootCAPath can be nil, because it is optional.
if option is not set, the client runs into nil pointer exception.
please add nil handling for config.RootCAPath, see test pipeline

Reply:
@shearn89 please implement this and after that we can merge and release your feature